samples/archivist_story_runner.py

```diff
@@ -0,0 +1,305 @@
+"""archivist story runner takes a yaml config file listing operations to be conducted serially.
+
+It then runs the defined operations.
+"""
+
+import time
+from enum import Enum
+import argparse
+import yaml
+
+# remove warning from console
+import urllib3
+
+from archivist.compliance_polices import PolicyType
+from archivist.archivist import Archivist
+
+# stop printing to console that the request is insecure
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+class Operation(Enum):
+    """
+    Operation are the allowed archivist operations to be run.
+    """
+
+    CREATE_ASSET = 1
+    CREATE_EVENT = 2
+    CREATE_COMPLIANCE_POLICY = 3
+    CHECK_COMPLIANCE = 4
+    DELETE_COMPLIANCE = 5
+
+
+def parse_arg(arg, key, default=None):
+    """
+    parse_arg parses a dictionary arg, and returns the value of a key.
```

Comment:
defaultdict() is better here

https://docs.python.org/3.6/library/collections.html#collections.defaultdict

Reply:
have opted for `setdefault`, but if defaultdict is a better solution happy to use it :)